Update/ura_005.txt

```diff
@@ -3799,14 +3799,14 @@ void main()
 			NULL, " After all, this case is so out of the blue, and not enough of the pieces fit together.", Line_WaitForInput);
 	ModPlayVoiceLS(4, 31, "ps3/s14/31/vt4bb_fujita1044", 256, TRUE);
 	OutputLine(NULL, "むしろ祟りならどれだけ考えるのが楽か、なんて思いたくもなりますよ」",
-			NULL, " But it's much easier to think it's the work of a curse.\"", Line_Normal);
+			NULL, " So there have been times where I've thought it would be so easy if I could assume it's the work of a curse.\"", Line_Normal);
```

Comment:
```suggestion
			NULL, " So there have been times where I've thought it would be so easy if I could just assume it's the work of a curse.\"", Line_Normal);
```